lib/command/run-workers.js

```diff
@@ -59,6 +60,16 @@ module.exports = function (workers, options) {
   output.print(`Running tests in ${output.styles.bold(workers)} workers...`);
   output.print();
 
+  let outputDir;
+
+  if (path.isAbsolute(config.output)) outputDir = config.output;
```

Comment:
I think, better to use curly brackets blocks.
A few added strings will improve readability without ant problems.

https://github.com/airbnb/javascript#blocks--braces